Docker_Weaviate.ipynb

```diff
@@ -0,0 +1,236 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Semantic Search using Weaviate and Docker "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial we create a vector store that can be queried using semantic search on a sample dataset composed of transcribed podcasts. The steps will include uploading your data from a local store, and creating a schema as well as an object store.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In your terminal:  \n",
+    "1. Run your virtual environment:  conda activate /Users/your_path/environment_name OR source path_to_your_VR/bin/activate\n",
+    "2. Download and run the yml image doc in this repo\n",
+    "3. Run docker-compose up -d\n",
+    "4. Run pip install -r requirements.txt"
```

Comment:
Did you create a requirements.txt? It would be nice if you could commit it as well.